Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+@Library('releng-pipeline') _
+
+hugo (
+  appName: 'eclipse.org/opensmartclide',
+  productionDomain: 'eclipse.org/opensmartclide',
```

Comment:
I added this here because this is what I think is the production domain but we will have to confirm.

Reply:
Project websites can't use eclispefdn-webdev project on OKD.

Reply:
I made this PR to unblock @phkrief 

https://github.com/eclipse-opensmartclide/opensmartclide-website.src/pull/5/files

Reply:
Sounds good, thanks.